eessi_bot_event_handler.py

```diff
@@ -21,13 +21,22 @@
 from connections import github
 from tools import config
 from tools.args import event_handler_parse
-from tasks.build import check_build_permission, submit_build_jobs, get_repo_cfg
+from tools.commands import get_bot_command, EESSIBotCommand, EESSIBotCommandError
+from tools.filter import EESSIBotActionFilter
+from tools.permissions import check_command_permission
+from tools.pr_comments import update_pr_comment
+from tasks.build import check_build_permission, submit_build_jobs, get_repo_cfg, get_architecturetargets
 from tasks.deploy import deploy_built_artefacts
 
 from pyghee.lib import PyGHee, create_app, get_event_info, read_event_from_json
 from pyghee.utils import log
 
 
+GITHUB = "github"
+APP_NAME = "app_name"
```

Comment:
These are also defined in `task/build.py`, which suggests there should be a common `constants` module to import these from?

Can be done in a follow-up PR of course